src/main/resources/application.yml

```diff
@@ -1,47 +1,73 @@
 spring:
-  datasource:
```

Comment:
yml files should be formatted with two spaces rather than four, please could you convert back